package.json

```
@@ -5,11 +5,12 @@
   "main": "index.js",
   "scripts": {
     "build": "tsc",
+    "clean": "rm -rf node_modules tscsonfig.tsbuildinfo dist",
```

To try and maintain support with Windows outside of WSL, might we use?:

```suggestion
    "clean": "git clean -dfqX -- ./node_modules ./tscsonfig.tsbuildinfo ./dist",
```

We do this elsewhere, but I'm curious if it matters.  Is that worth trying?

---

done!